treeherder/webapp/api/performance_data.py

```diff
@@ -426,9 +426,9 @@ def list(self, request):
         signature_data = (PerformanceSignature.objects
                                               .select_related('framework', 'repository', 'platform', 'push', 'job')
                                               .filter(repository__name=repository_name))
-
-        if signature:
-            signature_data = signature_data.filter(id=signature)
+        print(signature)
```

Comment:
Is this ``print`` a leftover from debugging?

Reply:
hah, yes!